executor/README.MD

```diff
@@ -0,0 +1,108 @@
+For example, to build the executable, run the following command from the `executor` folder:
+
+```bash
+# will create dist/cook-executor
+$ pyinstaller -F -n cook-executor -p cook cook/__main__.py
+```
+
+For development, a docker-compatible version of the cook executor can be built using:
+
+```bash
+# will create dist/cook-executor
+$ bin/build-cook-executor.sh
+```
+
+### Configuration
+
+Configuration for the executor should go under the `:executor` key of the edn configuration file for the cook scheduler.
+
+For example:
+
+```clojure
+{...
+ :executor {:command "cook-executor"
+            :log-level "INFO"
+            :max-message-length 512
+            :progress-output-name "stdout"
+            :progress-regex-string "\^\^\^\^JOB-PROGRESS: (\d*)(?: )?(.*)"
+            :progress-sample-interval-ms 1000
+            :uri {:cache true
+                  :executable true
+                  :extract false
+                  :value "file:///path/to/cook-executor"}}
+ ...}
+```
+
+Supported configuration options:
+
+| option | type | description |
+|--------|------|-------------|
+| `:command` | string | A string containing the command executed on the mesos agent to launch the cook executor. If the executor is installed using the instructions above, the default value of `cook-executor` should not need to be changed. |
+| `:log-level` | string | The log level for the executor process. Defaults to "INFO" |
+| `:max-message-length` | long | The maximum length for the custom failure message set by a task via the Mesos executor HTTP API. The default is 512. |
+| `:progress-output-name` | string | The file to track for progress updates. The default is stdout. |
+| `:progress-regex-string` | string | The regex used to identify progress update messages. The regex should generates two components in the match, the first being an integer representing the progress percent. The second being a message about the progress. When not provided, progress updates will not be tracked. |
+| `:progress-sample-interval-ms` | long | The interval in ms after which to send progress updates. The default is 1000. |
+| `:uri` | map | A map of the `uri` to be associated with every the job. |
+
+`uri`s have the following structure:
+| key | type | description |
+|-----|------|-------------|
+| `:cache` | boolean | Mesos 0.23 and later only: should the URI be cached in the fetcher cache? |
+| `:executable` | boolean | Should the URI have the executable bit set after download? |
+| `:extract` | boolean | Should the URI be extracted (must be a tar.gz, zipfile, or similar).
+| `:value` | string | The URI to fetch. Supports everything the Mesos fetcher supports, i.e. http://, https://, ftp://, file://, hdfs:// |
+
+### Tests
+
+The cook executor uses `nose`.
+To install test dependencies and run the executor test suite, run:
+
+```bash
+$ python setup.py nosetests
```

Comment:
Would just `nosetests` suffice? I believe that's what we have documented for the integration tests